pkgs/development/python-modules/magic-wormhole/default.nix

```diff
@@ -2,37 +2,38 @@
 , buildPythonPackage
 , fetchPypi
 , pythonAtLeast
-, nettools
-, glibcLocales
+, python
+, spake2
+, pynacl
+, six
+, attrs
+, twisted
 , autobahn
-, cffi
-, click
+, automat
 , hkdf
-, pynacl
-, spake2
 , tqdm
-, python
-, mock
-, ipaddress
+, click
 , humanize
-, pyopenssl
-, service-identity
+, ipaddress
 , txtorcon
+, nettools
+, glibcLocales
+, mock
+, magic-wormhole-transit-relay
 }:
 
 buildPythonPackage rec {
   pname = "magic-wormhole";
   version = "0.10.5";
-  name = "${pname}-${version}";
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "9558ea1f3551e535deec3462cd5c8391cb32ebb12ecd8b40b36861dbee4917ee";
   };
 
-  checkInputs = [ mock ];
+  checkInputs = [ mock magic-wormhole-transit-relay ];
   buildInputs = [ nettools glibcLocales ];
-  propagatedBuildInputs = [ autobahn cffi click hkdf pynacl spake2 tqdm ipaddress humanize pyopenssl service-identity txtorcon ];
+  propagatedBuildInputs = [ spake2 pynacl six attrs twisted autobahn automat hkdf tqdm click humanize ipaddress txtorcon ];
```

Comment:
Sorry that I changed the order so much. I just wanted the dependency to be in the same order as in `setup.py`.

Reply:
Thanks for the info. In general though, I think it's a much better idea to have these comments in the commit messages, so that people could find out without having to find this PR.

Reply:
Changed commit message.